flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why excluding aarch64 users?

Reply:
I'm not sure if all dependencies are working on `aarch64` so I wanted to not include it at first